lib/std/Target.zig

```diff
@@ -1602,6 +1602,165 @@ pub const Cpu = struct {
                 else => ".X",
             };
         }
+
+        /// Returns the array of `Arch` to which a specific `std.builtin.CallingConvention` applies.
+        /// Asserts that `cc` is not `.auto`, `.@"async"`, `.naked`, or `.@"inline"`.
+        pub fn fromCallconv(cc: std.builtin.CallingConvention.Tag) []const Arch {
```

Comment:
Nit: "callconv" spelling should be consistent one way or the other with `defaultCCallingConvention()` below.

I'm also a bit skeptical about placing both of these functions in `std.Target`; it feels like they're just a bit too coupled to the Zig compiler implementation to belong here. See e.g. my comment below about the ELFv2 hack for a quirk that a general user of this API would likely *not* want. Personally I think I'd place these in `src/target.zig` or similar, FWIW.

Reply:
> Nit: [...]

Good point!

> I'm also a bit skeptical about placing both of these functions in `std.Target`; it feels like they're just a bit too coupled to the Zig compiler implementation to belong here.

The concepts implemented by these two functions should be implementation-agnostic. I'll talk about this when responding to your next comment!